static/css/page-signup.less

```diff
@@ -153,9 +153,16 @@
     color: @grey;
   }
 
+  .input {
+    position: relative;
+  }
+
   .password-visibility-toggle {
-    float: right;
-    color: @grey;
+    position: absolute;
```

Comment:
instead of making the img position:absolute, make the containing <a> that. If you tab around you'll see the focus outline is funky.